### ISSUE TITLE
Automatic Docker image builds

### ISSUE THREAD

**Issue by Rzeszow** [meepen/salien-bot#73]
Are automatic docker images already available?
If not, I would ask for setting this.

**Comment by tokkenno** [comment]
Yes, use the image: https://hub.docker.com/r/reimashi/salien-bot/
